src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
Should not we use `ClassName[]`, `string[]` or `object` as values then? Because they are not valid "types" either.